### PR TITLE
feat(nasha): add @ovh-ux/manager-nasha module

### PR DESCRIPTION
**Pull request by varun257**
Create new module for nasha
Move code from cloud manager to standalone nasha module
Fix all dependencies
Migrate autocomplete module to cloud universe components
Use cuc autocomplete in nasha

Issue: MANAGER-3761

Signed-off-by: Varun <varun.shivaprasad@corp.ovh.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#7025.0 ✘  
* 
  * Build ✘

File: packages/manager/modules/nasha/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "lodash": "^4.17.15"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^7.5.0",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.3.2",
+    "@ovh-ux/ng-ovh-doc-url": "^1.0.0",
```
Comment (antleblanc) [inline packages/manager/modules/nasha/package.json]: No mentioned in the standalone application.

File: packages/manager/modules/nasha/src/partition/access/add/styles.less [hunk 2]
```diff
@@ -1,3 +1,7 @@
+@cloud_info_bg_color: #dfeaf8;
```
Comment (antleblanc) [inline packages/manager/modules/nasha/src/partition/access/add/styles.less]: Theses variables couldn't be scoped inside the `.nasha-partition-access-add` selector?

File: packages/manager/modules/nasha/src/routing.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      controller: 'NashaCtrl',
+      controllerAs: 'NashaCtrl',
+      translations: {
+        value: ['../common', '.'],
+        format: 'json',
+      },
+    })
+    .state('nasha-order-complete', {
+      url: '/nasha/order/complete',
+      templateUrl: 'app/nasha/order/nasha-order-complete.html',
```
Comment (antleblanc) [inline packages/manager/modules/nasha/src/routing.js]: All `templateUrl` couldn't be turned into a regular import statement?

**Comment by ovh-cds** [comment]
CDS Report installation#7180.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#7180.1 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#7180.2 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#7204.0 ■   
* Stage 1
  * Install ■

**Pull request merged**